### PR TITLE
start of project view and task approval

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

File: pages/project/formatters/index.jsx [hunk 1]
```diff
@@ -19,9 +34,18 @@ const formatters = {
   updatedAt: {
     format: date => format(date, dateFormat.datetime)
   },
-  version: {
-    format: ({ id }) =>
-      <Link page="project.version" versionId={id} label={'Current draft'} />
+  granted: {
+    format: (granted, model) => {
+      console.log(granted, model);
```
Comment (wheelsandcogs) [inline pages/project/formatters/index.jsx]: leaving this in?

Comment (joefitter) [reply pages/project/formatters/index.jsx]: OOPS

File: pages/project/read/index.js [hunk 2]
```diff
@@ -8,15 +8,26 @@ module.exports = settings => {
 
   app.use((req, res, next) => {
     req.breadcrumb('project.read');
+    console.log('HI FROM PROJ');
```
Comment (wheelsandcogs) [inline pages/project/read/index.js]: Oh HAI!

**Pull request merged**